hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/container/ListSubcommand.java

```diff
@@ -37,8 +37,18 @@ public class ListSubcommand implements Callable<Void> {
   @CommandLine.ParentCommand
   private ContainerCommands parent;
 
+  @CommandLine.Option(names = {"--path", "-p"},
```

Comment:
I meant that  we need to change the command description itself 
``` java
@Command(
    name = "list",
    description = "Show container info of all container replicas on datanode on a given DN storage directory. If the path option is not passed , it will consider the first DN storage directory from the configured values in hdds.datanode.dirs")
public class ListSubcommand implements Callable<Void> {
```
Do this for all container commands as that is what is followed.

Reply:
The command works, but we need to specify the path before the `list` subcommand, if we add the --path option in the ContainerCommands class. 

```
sh-4.2$ ozone debug container list --path=/data/hdds
Unknown option: '--path=/data/hdds'
Usage: ozone debug container list
Show container info of all container replicas on datanode
```

However, this works:

```
sh-4.2$ ozone debug container --path=/data/hdds list
{
  "containerType" : "KeyValueContainer",
  "containerID" : 1,
  "metadata" : { },
  "chunksPath" : "/data/hdds/hdds/CID-9a218a31-0faa-4588-95e1-fed536eb0f5c/current/containerDir0/1/chunks",
  "state" : "OPEN",
}
```

Reply:
There is no issue with passing the argument like this 
`ozone debug container --path=/data/hdds list ` . Why do you think this is a problem?

Reply:
Ok so I thought It reads more like natural language, saying "list containers at this path." 

But I agree Here, --path is placed right after container, which makes more sense because it’s a shared option for all subcommands (like list, info, etc.). This way, the path is applied globally before the specific subcommand is executed.

I'll make changes as suggested.

Reply:
`ozone debug ldb` commands also has a similar structure for --db option. I too believe it would be better to define the option in the parent command, instead of in each of the subcommands.

Reply:
Each of these container commands make operations relative to a path to storage directory , hence it makes sense to generalize the container commands cli itself and modify all the command descriptions to communicate this clearly

Reply:
> The command works, but we need to specify the path before the `list` subcommand, if we add the --path option in the ContainerCommands class.

Please check https://picocli.info/#_inherited_options